src/discrete-layer/components/map-container/map-container.tsx

```diff
@@ -130,32 +138,89 @@ export const MapContainer: React.FC<MapContainerProps> = (props) => {
 
   return (
     <div className="map">
-      <div className="filtersPosition" style={{backgroundColor: theme.primary, width: props.mapActionsWidth}}>
+      <div className="filtersPosition" style={{backgroundColor: theme.primary, width: mapActionsWidth}}>
         <div className="filtersContainer">
           <PolygonSelectionUi
             onCancelDraw={onCancelDraw}
             onReset={onReset}
             onStartDraw={setDrawType}
             isSelectionEnabled={isDrawing}
             onPolygonUpdate={onPolygonSelection}
-            mapActionsWidth = {props.mapActionsWidth}
-            handleOtherDrawers={props.handleOtherDrawers}
+            mapActionsWidth = {mapActionsWidth}
+            handleOtherDrawers={(): void => setFiltersOpen(false)}
           />
-          {props.filters?.map((filter, index) => (
-            <div key={index} className="filtersMargin">
-              {filter}
-            </div>
-          ))}
+          <div className="filtersMargin">
```

Comment:
is filters and table component should be part of file named "map container"?

Reply:
At least for a while, previously it was transferred from outside